### ISSUE TITLE
static or dynamic scheduling in simps_parallel_chunked

### ISSUE THREAD

**Issue by rachka** [uwhpsc-2016/homework3#16]
For the report and `simps_parallel_chunked` should we use static scheduling? The chunk size is given as an argument and I figure that everyone's reports should use the same scheduling for consistency. I assumed static scheduling in my code for `simps_parallel_chunked`. Is that correct?

**Comment by quantheory** [comment]
I don't believe that it matters. Using static scheduling is fine, though that doesn't prevent you from experimenting with dynamic scheduling if you're curious how it changes things.

**Comment by cswiercz** [comment]
See #30

**Comment by philw16** [comment]
I think we might have touched on this (or were going to touch on this), but this was an interesting read that talks some about load balancing impacts.

http://stackoverflow.com/questions/10850155/openmp-for-schedule
